internal/cli/base.go

```diff
@@ -24,6 +25,11 @@ import (
 	"github.com/hashicorp/waypoint/internal/server/grpcmetadata"
 )
 
+const (
+	defaultWorkspace        = "default"
+	defaultWorkspaceEnvName = "WP_WORKSPACE"
```

Comment:
This should probably be `WAYPOINT_WORKSPACE` to fit the pattern of our other env vars.
```suggestion
	defaultWorkspaceEnvName = "WAYPOINT_WORKSPACE"
```